touchforms/formplayer/api.py

```diff
@@ -283,10 +284,8 @@ def post_data(data, auth=None, content_type="application/json"):
     if auth:
         d['hq_auth'] = auth.to_dict()
     domain = d.get("domain")
-    d['restoreAs'] = d['username']
-
-    if domain:
-        d['uses_sql_backend'] = use_sqlite_backend(domain)
```

Comment:
Hm, good point - I'd convinced myself that this wasn't needed anymore, but looking again I'm not sure. Reverting

Reply:
@gcapalbo OK I remembered my logic on this. I'm pretty sure this would never be set to `True` for SMS requests because they weren't including the `domain` in the request payload until I [added that](https://github.com/dimagi/touchforms/pull/292/files/8107d2fbf2602075c487e13a7213b1735a0af703#diff-a8233683af9d9f2533f234c6e299320aR99) in this PR. So this would never have been set since SMS is the only thing still using this API. Including the domain and having this be set to `True` now causes problems. Let me know if you disagree with that logic.